manifests/init.pp

```diff
@@ -1,22 +1,28 @@
-class networkmanager(
-  $version = present,
-  $enable  = true,
-  $start   = true,
-  $gui     = undef,
+# Class networkmanager
+class networkmanager (
+  $manage_packages           = $networkmanager::params::manage_packages,
+  $package_ensure            = $networkmanager::params::package_ensure,
+  $manage_service            = $networkmanager::params::manage_service,
+  $service_enable            = $networkmanager::params::service_enable,
+  $service_ensure            = $networkmanager::params::service_ensure,
+  $gui                       = $networkmanager::params::gui,
+  $openconnect_connections   = {},
+  $openvpn_connections       = {},
+  $wifi_connections          = {},
+) inherits networkmanager::params {
 
-  $openconnect_connections = {},
-  $openvpn_connections     = {},
-  $wifi_connections        = {},
-) {
+  include ::stdlib
```

Comment:
Two reasons.
1. validate_re() is from stdlib
2. I'd like to use pick() to prefer $version, $enable, $start (vs. $package_ensure, $service_enable, $service_ensure)